src/content/docs/build/set-up-options/sync-git-code.mdx

```diff
@@ -0,0 +1,27 @@
+---
+page_id: 143daf4f-bc5b-4ceb-8af5-df83463fc9f5
+title: Manage your code in Kinde
+sidebar:
+  order: 6
+relatedArticles:
+  - 85c135db-f55a-4edf-8d1b-0af4140c1925
+  - 8554d648-fc2f-4ef9-aaa9-4f27b3c05205
+---
+
+To use the workflows and other BYO code features, you need to sync your git-stored code to Kinde. Currently, Kinde only syncs with Github, but other repo types will be added as the features progress.
```

Comment:
@clairekinde11 
Todo we need “the” in “the workflows”?

I think saying “other providers” rather than “repo types”